src/jdk.jpackage/macosx/classes/jdk/jpackage/internal/AppImageSigner.java

```diff
@@ -58,8 +60,10 @@ static Consumer<MacBundle> createSigner(MacApplication app, CodesignConfig signi
             try {
                 new AppImageSigner(Codesigners.create(signingCfg)).sign(app, appImage);
             } catch (CodesignException ex) {
+                LOGGER.log(Level.ERROR, ex);
```

Comment:
This log statement is redundant. There is no point in logging exceptions being passed through.

This catch block catches an exception, passes it to `handleCodesignException()` method, and then rethrows it. What additional information do we need about this "event"?